packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -525,6 +526,41 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
 /// of [Curves.fastOutSlowIn].
 /// {@animation 250 266 https://flutter.github.io/assets-for-api-docs/assets/widgets/animated_container.mp4}
 ///
+/// {@tool snippet --template=stateful_widget_scaffold}
```

Comment:
Somehow (I don't think we have figured out how to do that yet) we should link the example code with the animation to make it clear to people that the animation is illustrating what the code is showing. Not sure how to best do that. Ideas:

* Embed the animation within the `@tool snippet` section
* Say with words in the example description below that the resulting animation can be seen above (and vice versa)
* ...

Maybe @gspencergoog has a good idea.

Reply:
I like embedding the animation within the description of the example code for now.  Eventually, it would be nice to have a side-by-side presentation, or a separate tab with the animation in it, but for now that doesn't exist.

Reply:
Previous samples & diagrams/animations have embedded the assets in the description.

Reply:
Do you happen to know of an example that does that? Just curious what it looks like rendered out in dart docs.

Reply:
I don't think we're quite set up to do that yet. The white-on-blue doesn't quite look right, and the video comment ends up being generated in the full snippet:

![Screen Shot 2019-07-09 at 10 59 50 AM](https://user-images.githubusercontent.com/1145719/60912100-7ddd4700-a239-11e9-8132-c9aabc451397.png)
![Screen Shot 2019-07-09 at 11 00 06 AM](https://user-images.githubusercontent.com/1145719/60912101-7ddd4700-a239-11e9-8e94-15630d9c38fe.png)

Reply:
for now I added a "depicted above" to the description

Reply:
Here's an example: https://master-api.flutter.dev/flutter/widgets/Flow-class.html
I agree the white-on-blue doesn't appear very polished:
![Screen Shot 2019-07-09 at 12 41 21 PM](https://user-images.githubusercontent.com/16964204/60917992-02829200-a247-11e9-9c77-c14dadbde220.png)